specification/translation/Azure.AI.DocumentTranslation/routes.tsp

```diff
@@ -0,0 +1,460 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Versioning;
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace DocumentTranslation;
+
+alias ServiceTraits = SupportsRepeatableRequests &
+  SupportsConditionalRequests &
+  SupportsClientRequestId;
+
+interface SingleDocumentTranslationOperations {
+  #suppress "@azure-tools/typespec-azure-core/byos"
+  @summary("Translate a single document")
+  @doc("API to translate single document")
+  @post
+  @route("document:translate")
+  singleDocumentTranslate is RpcOperation<
```

Comment:
If this is the operation that was added in the new preview version please use the `@added` decorator over it and indicate what api version is adding this. Here's a link to the docs: https://typespec.io/docs/libraries/versioning/reference/decorators#@TypeSpec.Versioning.added

Reply:
We decided to go with a single version